exercises/meetup/test.ml

```diff
@@ -0,0 +1,786 @@
+open OUnit2
+open Meetup
+open Core.Std
+
+type case = {
+  description: string;
+  year: int;
+  month: int;
+  week: schedule;
+  dayofweek: weekday;
+  dayofmonth: int
+}
+
+let ae exp got _test_ctxt = assert_equal
+  ~printer:(fun (y, m, d) -> sprintf "%d-%02d-%02d" y m d) exp got
+
+let make_test (c: case) =
+  c.description >::
+    ae (c.year, c.month, c.dayofmonth)
+       (meetup_day c.week c.dayofweek ~year:c.year ~month:c.month)
+
+let tests = List.map ~f:make_test [
```

Comment:
Thank you Daan!! I can take no credit for the way the tests are written - I found the data driven idea in the xhaskell meetup tests.